Microsoft.Azure.Cosmos/src/Handler/RequestInvokerHandler.cs

```diff
@@ -431,6 +436,29 @@ private async Task ValidateAndSetConsistencyLevelAsync(RequestMessage requestMes
             }
         }
 
+        /// <summary>
+        /// Set the PriorityLevel in the request headers
+        /// </summary>
+        /// <param name="requestMessage"></param>
+        private void SetPriorityLevel(RequestMessage requestMessage)
+        {
+            Cosmos.PriorityLevel? priorityLevel = null;
```

Comment:
If the variable is `Cosmos.PriorityLevel?` might as well initialize it with:

```suggestion
            Cosmos.PriorityLevel? priorityLevel = this.RequestedClientPriorityLevel;
```

This avoids the `else if` clause below.

Reply:
Changed as suggested